src/scripts/main.js

```diff
@@ -1,11 +1,31 @@
+/* eslint-disable prettier/prettier */
 'use strict';
 
 function waitFor(element, eventName) {
-  // write your code here
+  const promise = new Promise((resolve) => {
+    element.addEventListener(eventName, (e) => {
+      // eslint-disable-next-line no-undef
+      resolve(`It was ${eventName} on the element: ${element.nodeName}, id: ${element.id}.`);
```

Comment:
Avoid using template strings for simple concatenations. This can make your code less readable and more prone to errors. Instead, use the '+' operator to concatenate strings.